storage/src/mmr/verification.rs

```diff
@@ -778,14 +793,15 @@ mod tests {
             let end_pos = *element_positions.last().unwrap();
             let range_proof = mmr.range_proof(start_pos, end_pos).await.unwrap();
             assert!(
-		range_proof.verify_range_inclusion(
+                range_proof.verify_range_inclusion(
                     &mut hasher,
-                    &elements,
+                    elements,
                     start_pos,
                     end_pos,
                     &updated_root_hash,
-		),
-		"valid range proof over remaining elements after 2 pruning rounds should verify successfully",
+                ),
+                "valid range proof over remaining elements after 2 pruning rounds should verify \
```

Comment:
I fear this `\` is included in the string? Not sure.

Reply:
it's not, but reverted anyway.  I put this in there because the long line seemed to piss off vscode and it would no longer auto-format-on-save this section.